tests/integration/suite/daprd/state/http/fuzz.go

```diff
@@ -153,7 +155,10 @@ func (f *fuzzstate) Run(t *testing.T, ctx context.Context) {
 	t.Run("get", func(t *testing.T) {
 		t.Parallel()
 		for i := range f.getFuzzKeys {
-			getURL := fmt.Sprintf("http://localhost:%d/v1.0/state/%s/%s", f.daprd.HTTPPort(), url.PathEscape(f.storeName), url.PathEscape(f.getFuzzKeys[i]))
+			getURL := fmt.Sprintf("http://localhost:%d/v1.0/state/%s/%s", f.daprd.HTTPPort(), url.QueryEscape(f.storeName), url.QueryEscape(f.getFuzzKeys[i]))
+			t.Log("URL", getURL)
+			t.Log("State store name", f.storeName, hex.EncodeToString([]byte(f.storeName)), printRunes(f.storeName))
+			t.Log("Key", f.getFuzzKeys[i], printRunes(f.getFuzzKeys[i]))
```

Comment:
I don't think we should add logs here. This massively balloons the output. Ideally if something errors, all information will be included in the `assert`/`require` output.

Reply:
These logs should appear only if the test fails? I can comment them out if needed.

Reply:
It's just because if something significant is wrong it balloons the logs and makes text editors/terminals sluggish.

Reply:
Ok I can comment those out. I would like to leave them in case they are useful in the future to debug more issues.

Reply:
Done